doc/source/parameters/cfd/dynamic_flow_control.rst

```diff
@@ -1,39 +1,61 @@
 Dynamic flow control
 ~~~~~~~~~~~~~~~~~~~~
 
-The purpose of this subsection is to enable dynamic flow control. It is important when we want to target a volumetric flow on a specific boundary. To control the volumetric flow, the code calculates a :math:`\beta`  coefficient at each time step that is used to keep the volumetric flow value inside an acceptable threshold. If the chosen wall boundary on which a volumetric flow is imposed is an inlet, the volumetric flow rate targeted must be negative since the outward normal vector is used for the calculation. Thereby, if the chosen wall boundary is an outlet, volumetric flow rate must be positive.
+The purpose of this subsection is to enable dynamic flow control. It is important when we want to simulate an average
+velocity on a specific boundary (CFD) or the whole domain (for CFD-DEM). To control the average velocity of the flow, the code
+calculates a :math:`\beta` coefficient at each time step that is used to keep the average velocity in an acceptable
+threshold. If the chosen wall boundary on which a average velocity is imposed is an inlet, the average velocity
+targeted must be negative since the outward normal vector is used for the calculation. Thereby, if the chosen wall
+boundary is an outlet, average velocity must be positive. In other to keep the sign consistency, the targeted
+velocity should have an negative sign when used with CFD-DEM even if no normal vector is used since the
+whole domain is taken into account.
+
+The main controller of the average velocity is the following equation and is based on Wang [1]:
```

Comment:
```suggestion
The main controller of the average velocity is the following equation and is based on approach of Wang [1]:
```

Reply:
et al? are there many authors to the Wang paper?

Reply:
This is his/her thesis

Reply:
great